index.js

```diff
@@ -7,7 +7,8 @@
   var xmlParser = require('xml2json');
   var FundamentalAccountingConcepts = require('./FundamentalAccountingConcepts.js');
 
-  function parse(pathToXbrlDoc) {
+  function parse(pathToXbrlDoc, output) {
+    output = (output=== null)? false : output;
     var self = this;
```

Comment:
so I set it to 
output = output === true;
which will set everything to false, unless true is passed in